scripts/deployToken.js

```diff
@@ -0,0 +1,46 @@
+const hre = require('hardhat')
+const ethers = hre.ethers;
+const fs = require('fs');
+
+const GAS_LIMIT = '8000000'
+
+async function deployToken() {
+
+    await hre.run('compile');
+    const [deployer] = await ethers.getSigners();
+
+    console.log('Deploying contract with the account:', deployer.address);
+    console.log('Account balance:', (await deployer.getBalance()).toString());
+
+    const name = '$ICONS Token'
+    const symbol = '$ICONS'
+    const totalSupply = ethers.utils.parseEther("30000000")
+    // const gnosisSafeWallet = '0x9692220eE7424c8A3FAE7df7057e4d02296bE075'
```

Comment:
this should be uncommented. The most recent gnosis we are using is `0xdd863aE1970938294190A435f0758A8d4a62031A`.